### ISSUE TITLE
Template for feature requests

### ISSUE THREAD

**Issue by onurcakmakceng** [getanteon/gurubase#19]
Nice to have a template for feature requests

**Comment by kursataktas** [comment]
Yes, we should have. I also suggest that you join our [Discord channel](https://discord.gg/9CMRSQPqx6) for quick requests.
